mapeditor/generatorprogress.cpp

```diff
@@ -0,0 +1,38 @@
+#include "StdInc.h"
+#include "generatorprogress.h"
+#include "ui_generatorprogress.h"
+#include <thread>
+#include <chrono>
+
+GeneratorProgress::GeneratorProgress(Load::Progress & source, QWidget *parent) :
+	QDialog(parent),
+	ui(new Ui::GeneratorProgress),
```

Comment:
Can we use std::unique_ptr?

Reply:
no no, it's qt stuff

Reply:
well, it's not a problem to use unique_ptr here. It's just that Qt Creator's template creates naked pointer plus deletes it in destructor. I'd keep as is.

Reply:
Yes sure if it is qt approach